docs/development.md

```diff
@@ -62,6 +62,22 @@ dates to help avoid unintentially running test pipelines over large amounts of
 data. For appending pipelines this will be the latest week of data, and for
 full pipelines it will be the previous week of data.
 
+## Running queries
+
+After recreating the base tables you can rebuild any derived tables by running
+
+    python3 -m table.run_queries
+
+To test changes to the queries without overwriting the derived tables write to
+a personal dataset like
+
+    python3 -m table.run_queries --sink=<username>
```

Comment:
I'd use  `input` and `output` instead of `source` and `sink`. I believe it's more common and clearer. Or perhaps we can be more explicit and say `base_dataset` and `derived_dataset`

Reply:
done